api/migrations/0012_auto_20170108_1908.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
is this file supposed to be part of this commit/PR? seems related to the day of week stuff that was happening elsewhere